bin/ch/Debugger.cpp

```diff
@@ -174,10 +174,7 @@ JsValueRef Debugger::Evaluate(JsValueRef callee, bool isConstructCall, JsValueRe
     if (argumentCount > 2)
     {
         IfJsErrorFailLogAndRet(ChakraRTInterface::JsNumberToInt(arguments[1], &stackFrameIndex));
-
-        AutoString argstr(arguments[2]);
-        IfJsErrorFailLogAndRet(argstr.GetError());
-        ChakraRTInterface::JsDiagEvaluateUtf8(argstr.GetString(), stackFrameIndex, &result);
+        ChakraRTInterface::JsDiagEvaluate(arguments[2], stackFrameIndex, JsParseScriptAttributeNone, &result);
```

Comment:
>JsDiagEvaluate [](start = 27, length = 14)

We need to add test for covering JsDiagEvaluate where 1st argument is ExternalArrayBuffer.
The way we have ch test debugger written Debugger::Evaluate is called from js file so argument is already JsValueRef (JavascriptString). One way to test external buffer is to add a new argument to evaluate which will convert string to external buffer.